actions/utils.js

```diff
@@ -121,12 +123,46 @@ function getDocPathFromUrl(url) {
     return `${path}.docx`;
 }
 
+async function updateStatus(projectExcelPath, status) {
+  const logger = getAioLogger();
+  // md5 hash of the config file
+  try {
+    const hash = crypto.createHash('md5').update(projectExcelPath).digest('hex');
+    logger.info(`Adding status to aio state lib with hash -- ${hash} - ${status}`);
+    const state = await stateLib.init();
+    state.put(hash, status);
```

Comment:
This is just pushing the status string, correct? How is it differentiated between copy status and promote status?

Reply:
projectRootPath -- milo vs milo-pink

Reply:
Not really. The project file XLSX is always under /milo.. not under /milo-pink. It should always be the same project whether it is a copy action or promote action.